run/helm/snap/charts/grafana/templates/job.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.server.setDatasource.enabled -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Values.server.name }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "server.fullname" . }}-set-datasource
+  annotations:
+    pod.beta.kubernetes.io/init-containers: '[
+        {
+            "name": "wait-for-grafana",
+            "image": "cosmintitei/bash-curl",
```

Comment:
Made configurable. I just needed Docker image with bash and curl. We can replace it later with our own image.